src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/ContainerRecord.java

```diff
@@ -48,6 +51,22 @@ public String getContainerId() {
     }
 
     /**
+     * Get the ID of the image from which this container was started.
+     * @return The ID of the image from which this container was started.
+     */
+    public String getImageId() {
+        return imageId;
+    }
+
+    /**
+     * Set the image ID of the image from which this container was started.
+     * @param imageId The image ID of the image from which this container was started.
+     */
+    public void setImageId(@Nonnull String imageId) {
```

Comment:
It would be if you wanted to make the equivalent of `DockerRunFingerprintFacet` somewhere else.